blueprints/hr_assistant/date.py

```diff
@@ -0,0 +1,323 @@
+# -*- coding: utf-8 -*-
+"""This module contains the dialogue states for the 'date' domain in
+the MindMeld HR assistant blueprint application
+"""
+import os
+import requests
+from .root import app
+from hr_assistant.general import _resolve_categorical_entities, _resolve_function_entity, _resolve_extremes, _agg_function, _get_names, _get_person_info, _fetch_from_kb, _not_an_employee
```

Comment:
This should be folded over multiple lines or just `from hr_assistant import general`